clang/lib/Driver/Driver.cpp

```diff
@@ -5181,19 +5148,12 @@ class OffloadingActionBuilder final {
     /// Does not track AOT binary inputs triples.
     SmallVector<llvm::Triple, 4> SYCLTripleList;
 
-    /// Type of output file for FPGA device compilation.
-    types::ID FPGAOutType = types::TY_FPGA_AOCX;
-
     /// List of objects to extract FPGA dependency info from
     ActionList FPGAObjectInputs;
 
     /// List of static archives to extract FPGA dependency info from
```

Comment:
Why has this been left behind?

Thanks

Reply:
This should be cleaned up now with my recent update.